include/solvers/mpm_base.tcc

```diff
@@ -497,13 +474,16 @@ void mpm::MPMBase<Tdim>::write_vtk(mpm::Index step, mpm::Index max_steps) {
   MPI_Comm_size(MPI_COMM_WORLD, &mpi_size);
 #endif
 
+  //! VTK vector variables
+  std::vector<std::string> vtk_vector_data = {"displacements", "velocities"};
```

Comment:
Should we add `accelerations` as the default?

Reply:
`accelerations` data is not stored at the particle level.

Reply:
My bad, yes!